packages/entities/entities-plugins/src/locales/en.json

```diff
@@ -19,7 +19,7 @@
   },
   "view_configuration": {
     "title": "Configuration",
-    "message": "Configuring your API has never been easier. Use YAML for human-friendly simplicity or JSON for machine-readability. Streamline your settings effortlessly.",
+    "message": "Export configuration of core Gateway Manager objects, like services, routes, and plugins, to YAML or JSON. This allows you to import the file to APIOps tooling, like decK.",
```

Comment:
These strings are potentially displayed in Kong Manager as well -- should we be mentioning `Gateway Manager`?

@jillztom

Reply:
Outside of Konnect, this `View Configuration` slideout is also consumed by Kong Manager and Kong Manager has no concept of  `Gateway Manager`, they used the term `Workspace` instead 
<img width="1724" alt="Screenshot 2024-02-02 at 2 39 16 PM" src="https://github.com/Kong/public-ui-components/assets/2568272/7fb47d26-5ea2-42ed-8324-d299bc2883ef">

Reply:
```suggestion
    "message": "Export configuration of core entities, like services, routes, and plugins, to YAML or JSON. This allows you to import the file to APIOps tooling, like decK.",
```